Gems/EMotionFX/Code/EMotionFX/Pipeline/RCExt/Actor/ActorGroupExporter.cpp

```diff
@@ -173,7 +173,7 @@ namespace EMotionFX
             }
 
             // Default to the first mesh group until we get a way to choose it via the scene settings (ATOM-13590).
-            AZ_Error("EMotionFX", atomModelAssets.size() <= 1, "Ambigious mesh for actor asset. More than one mesh group found. Defaulting to the first one.");
+            AZ_Warning("EMotionFX", atomModelAssets.size() <= 1, "Ambigious mesh for actor asset. More than one mesh group found. Defaulting to the first one.");
```

Comment:
Part of the problem is that the code is not actually checking for how many mesh groups there are any more. It's checking for how many .azmodels there are. It used to be there was a 1:1 correlation between mesh groups and azmodels, so the error about "more than one mesh group found" was accurate. Now, with procedural prefabs, counting the # of product models is no longer a valid way to count the # of mesh groups.

Is there a way at this point in the pipeline to still check the scene settings for the asset? That way you can get the scene settings, check the number of mesh groups, and have an error when there is actually more than one mesh group (which is ambiguous), but have no error at all with 1 mesh group and multiple product azmodels (which is valid, and not (as) ambiguous since the model you want is the only non-procedural prefab model that exists.

It's also worth verifying that the logic here of choosing the 'first' product is actually going to give you the atom model associated with the mesh group in the scene settings, and not one of the atom models that come from procedural prefabs.

Reply:
Is there a way to check if the atom model is created by the proc prefab?